Domain/Aggreagtes/StudentAggregate/Student.cs

```diff
@@ -16,18 +18,20 @@ public class Student : AuditableEntity, IAggregateRoot
         public DateOnly? DateOfBirth { get; private set; }
         public Address? Address { get; private set; }
         public EducationLevel? Education { get; private set; }
-
         public Guid? CourseId { get; private set; }
         public virtual Course? Course { get; private set; }
+        public Guid UserId { get; private set; } = default!;
+        public virtual User User { get; private set; }
```

Comment:
why the userId and User object in the student class used for? A student can exist without having a credentials.

Reply:
The inclusion of the UserId and User object in the Student class is to support scenarios where students need to authenticate and access the system.